src/components/toc.js

```diff
@@ -97,10 +96,10 @@ const itemAbove = item => {
 
 const lastTransitiveItem = item => {
 	const list = item.querySelector(`ul`)
-	if (!list) return item
+	if (!list || list.children.length === 0) return item
```

Comment:
Here (and in one other place) you used `?.`, which I didn't even know about 😮 and liked immediately. Will use that henceforth!

But this code here runs in the browser and [Firefox Mobile doesn't support `?.`](https://caniuse.com/mdn-javascript_operators_optional_chaining). 😭